src/track/trackmetadatataglib.cpp

```diff
@@ -1026,6 +1026,19 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
     const TagLib::ID3v2::FrameList bpmFrame(tag.frameListMap()["TBPM"]);
     if (!bpmFrame.isEmpty()) {
         parseBpm(pTrackMetadata, toQStringFirstNotEmpty(bpmFrame));
+        long bpmValue = pTrackMetadata->getBpm().getValue();
+        //Some software uses (or used) to write decimated values without comma,
+        //so the number reads as 1352 or 14525 when it is 135.2 or 145.25
+        if (bpmValue >= Bpm::kValueMinForTwoDecimals) {
+            qWarning() << " Changing BPM on " << pTrackMetadata->getArtist() << " - " << 
+                pTrackMetadata->getTitle() << " from " << bpmValue << " to " << bpmValue/100.0;
+            bpmValue /= 100.0;
+        } else if (bpmValue >= Bpm::kValueMinForOneDecimal) {
+            qWarning() << " Changing BPM on " << pTrackMetadata->getArtist() << " - " << 
+                pTrackMetadata->getTitle() << " from " << bpmValue << " to " << bpmValue/10.0;
+            bpmValue /= 10.0;
+        }
+        pTrackMetadata->getBpm().setValue(bpmValue);
```

Comment:
If we one day have the write back loop working this might change the ID3 tag probably unwanted. 
I think it is better to keep the original value, and use the tweaked value for use inside Mixxx only. 
@uklotzde: Or is this not an issue here? 

The two branches above can be turned into a while loop. 

```
while (bpmValue > Bpm::kValueMax) {
    bpmValue \= 10.0;  
}
```